e2e/cypress/integration/part.spec.js

```diff
@@ -28,13 +28,13 @@ describe("Part View", () => {
 
   it("loads a part view", () => {
     cy.visit("/433/");
-    cy.contains("433.51").click()
+    cy.findByRole("link", { name: "§ 433.51 Public Funds as the State share of financial participation." }).click()
 
     cy.get("a").contains("Part View").click();
 
-    cy.url().should("include", "#433-51");
-    cy.get("#433").contains("PART 433—STATE FISCAL ADMINISTRATION").should("be.visible");
```

Comment:
I like leaving this check in to make sure you are on the part page instead of subpart.